projects/admin-core/src/lib/application/state/application.reducer.ts

```diff
@@ -360,6 +360,79 @@ const onUpdateApplicationFiltersConfig = (
   }));
 };
 
+const onCreateApplicationFilterGroup = (
+  state: ApplicationState,
+  payload: ReturnType<typeof ApplicationActions.createApplicationAttributeFilter>,
+): ApplicationState => {
+  return updateApplication(state, application => {
+    let addedToExistingGroup = false;
```

Comment:
Check if this can be simplified by using a findIndex, then update the state accordingly. See for example `onUpdateApplicationComponentConfig`